lib/mongo/client.rb

```diff
@@ -446,10 +446,18 @@ def hash
     #       JSON Schemas obtained from the server. It protects against a
     #       malicious server advertising a false JSON Schema, which could trick
     #       the client into sending unencrypted data that should be encrypted.
+    #     - Note: If a collection is present on both the encryptedFieldsMap
```

Comment:
`in both the :encrypted_fields_map and :schema_map`

Reply:
Fixed